new/LIT60496MMIntrPrayer.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT60496MMIntrPrayer" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Introductory Prayer to the Miracles of Mary</title>
```

Comment:
OK, I added incipits to LIT4270Salota

Reply:
No, it is unlikely. The prayer LIT60496 is a liturgical one, with the deacon. LIT4270Salota is different, እግዚአብሔር፡  እግዚኦ፡  ኢየሱስ፡  ክርስቶስ፡  አምላክነ፡  ዘትቤሎ፡  ለፍቁርከ፡  ዮሐንስ፡  እንዘ፡  ሀሎከ፡  በዲበ፡  መስቀል፡  ቅዱስ፡  ወትቤላ፡  ለማርያም፡  ነዋ፡  ወልድኪ፡  ወለረድእከኒ፡ ነየ፡
The problem of tedious idenfitication will plague us unless we will include substantial portions of the text into work records. Otherwise it is like house with more and more stores, without foundation

Reply:
Oops, then I have to restore the record I had already deleted...

Reply:
But it is still the same prayer text, only differing in the introductive "liturgical directives". Here at least we have many manuscripts in which we can check the text. I would not be in favour of two IDs.

Reply:
Well, exactly introduction of "liturgical directives" make it different. It means - (better) adjusting the text for a new (liturgical) function which amounts to a new version. These slight changes have big implications.

Reply:
Possibly, then this would have to be clearly stated in the abstract. I also did not follow now which version of these records currently exists...
I think that there is a general "problem" with liturgical texts. There are what we might call individual prayers or hymns, as the one starting እግዚአብሔር፡ እግዚኦ፡ ኢየሱስ፡ ክርስቶስ፡ አምላክነ፡ ዘትቤሎ፡ ለፍቁርከ፡ ዮሐንስ፡ እንዘ፡ ሀሎከ፡ በዲበ፡ መስቀል፡ ቅዱስ፡ ወትቤላ፡ ለማርያም፡ ነዋ፡ ወልድኪ፡ ወለረድእከኒ፡ ነየ፡. Then these occur in manuscripts which contain a more detailed description of the liturgy, containing directives for the involved persons. Within the directives then, the individual prayers are stated (or, I think, sometimes also only referred to by their beginning, not quoted in full: "And then, the priest or whoever reads aloud salām la-ʾaba bərhānāt. And then, continue with...". 
There is a lot of confusion around all this. Shall there be one version for the text itself, and different ones for possible (diverging) liturgical introduction? Can we say that the liturgical directives are part of the prayer? 
Just to say that of course the only way forward is always to have more texts, and also explicits when available.